networks/remote/ansible/logzio.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- hosts: all
+  user: root
+  any_errors_fatal: true
+  gather_facts: no
+  vars:
+    - service: tendermint
+    - JOURNALBEAT_BINARY: "{{lookup('env', 'GOPATH')}}/bin/journalbeat"
+#    - LOGZIO_TOKEN: ABCDEFGHIJKLMNOPQRSTUVWXYZ012345
```

Comment:
leftover?

Reply:
Not really. I kept it there so people understand that it is necessary to run the playbook. I can remove it and add a comment on top to make sure they add it.